.github/workflows/optimize-e2e-test-cloud.out

```diff
@@ -0,0 +1,106 @@
+# NOTE:
+# This pipeline is currently not enabled. For more info, see:
+# https://github.com/camunda/camunda/issues/24886
+
+name: Optimize e2e cloud test
+
+on:
+  pull_request:
+    paths:
+      - ".github/actions/**"
+      - ".github/workflows/optimize-*"
+      - "bom/*"
+      - "parent/*"
+      - "pom.xml"
+      - "optimize/**"
+      - "optimize.Dockerfile"
+  push:
+    branches:
+      - "main"
+      - "stable/**"
+      - "release/**"
+    paths:
+      - ".github/actions/**"
+      - ".github/workflows/optimize-*"
+      - "bom/*"
+      - "parent/*"
+      - "pom.xml"
+      - "optimize/**"
+      - "optimize.Dockerfile"
+  workflow_dispatch:
+
+jobs:
```

Comment:
rather than commenting out the file, maybe a line like this with a comment would be sufficient:
```
# Some comment explaining why we disabled it
if: false
```